odl/operator/fn_ops.py

```diff
@@ -0,0 +1,406 @@
+## Copyright 2014-2016 The ODL development group
```

Comment:
Somehow snuck in dubble # here, remove one. Also it is 2017 now so might as well fix that while at it.